marklogic-client-api/src/main/java/com/marklogic/client/expression/CtsQueryBuilder.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright (c) 2019 MarkLogic Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.marklogic.client.expression;
+
+import com.marklogic.client.io.marker.JSONReadHandle;
+import com.marklogic.client.io.marker.JSONWriteHandle;
+import com.marklogic.client.query.CtsQueryDefinition;
+import com.marklogic.client.type.CtsQueryExpr;
+
```

Comment:
Not before checkin, but when you get a chance, adding JavaDoc for the public members on this and other public new classes will be important.

To be clear, implementations like AbstractSearchQueryDefinition don't need JavaDoc.

Reply:
I'll add Javadoc in my next check-in.